data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataexport/JsonLineExportManager.java

```diff
@@ -0,0 +1,43 @@
+package com.scalar.db.dataloader.core.dataexport;
+
+import com.scalar.db.api.DistributedStorage;
+import com.scalar.db.api.TableMetadata;
+import com.scalar.db.dataloader.core.dataexport.producer.ProducerTaskFactory;
+import com.scalar.db.dataloader.core.dataimport.dao.ScalarDBDao;
+import java.io.IOException;
+import java.io.Writer;
+
+public class JsonLineExportManager extends ExportManager {
+  public JsonLineExportManager(
+      DistributedStorage storage, ScalarDBDao dao, ProducerTaskFactory producerTaskFactory) {
+    super(storage, dao, producerTaskFactory);
+  }
+
+  /**
+   * Create and add header part for the export file
+   *
+   * @param exportOptions Export options for the data export
+   * @param tableMetadata Metadata of the table to export
+   * @param writer File writer object
+   * @throws IOException If any IO exception occurs
+   */
+  @Override
+  void processHeader(ExportOptions exportOptions, TableMetadata tableMetadata, Writer writer)
+      throws IOException {
+    System.out.println("SSSSSSSSSSSSSSSSSSSSSSSS");
```

Comment:
This is for debug?

Reply:
Yes, I forgot to remove it.
Removed it now.
Thank you.